content/projects/landspout.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Landspout
+repo: gmr/landspout
+homepage: https://github.com/gmr/landspout
+language:
+  - Python
+license:
+  - BSD 3-Clause
+templates:
+  - Tornado
+description: A simple static site generation tool
+---
+Landspout is a static website generation tool, using [Tornado Templates](http://www.tornadoweb.org/en/stable/). Create your template structure, and your content, and point landspout at it.
+
+Landspout has three operational modes:
```

Comment:
Everything below this can be removed since it's all in the readme. This page should only contain high level information.

Reply:
@gmr If you'd like to make that slight amendment suggested by @erquhart, I bet we could get this added to the site.

Reply:
Took care of it.